### PR TITLE
[UI] Fix configuration and services binding

### PR DESCRIPTION
**Pull request by torchiaf**
### Summary
Fixes https://github.com/epinio/ui/issues/219

We also fixed the binding of services and configurations of Application's upload from manifest, though the create & edit area need some regression tests

### CODE REVIEW

File: pkg/epinio/components/application/AppConfiguration.vue [hunk 1]
```diff
@@ -71,6 +70,11 @@ export default Vue.extend<Data, any, any, any>({
         .filter((s: EpinioService) => s.metadata.namespace === this.application.metadata.namespace);
     },
 
+    namespacedConfigurations() {
+      return (this.application.allConfigurations || [])
+        .filter((s: EpinioService) => s.metadata.namespace === this.application.metadata.namespace);
```
Comment (richard-cox) [inline pkg/epinio/components/application/AppConfiguration.vue:75]: `application.allConfigurations` already filters by namespace

**Review by richard-cox — CHANGES_REQUESTED** [review]
- Bug - Unable to bind non-service related configurations to an application when editing an application (they don't appear in list)

File: pkg/epinio/components/application/AppConfiguration.vue (hunk 1 above)
Comment (torchiaf) [reply pkg/epinio/components/application/AppConfiguration.vue:75]: The bug is coming from `allConfigurations` , where we are filtering based on namespace and configuration name from app's configuration:

```
.filter((s) => {
  return s.metadata.namespace === this.metadata.namespace &&
   this.configuration.configurations.find(c => c === s.metadata.name);
});
```

So just removing all configuration and filtering only by namespace.

Comment (richard-cox) [reply pkg/epinio/components/application/AppConfiguration.vue:75]: ahhh, sorry, i get it

**Pull request merged**